ui/job-view/PushHeader.jsx

```diff
@@ -23,7 +23,7 @@ const PushCounts = (props) => {
 
   return (
     <span className="push-progress">
-      {percentComplete === 100 ?
+      {total === 0 || percentComplete === 100 ?
```

Comment:
I wonder if we should display a third form of roughly "No jobs found" / "No jobs scheduled" / just blank?

Otherwise when the push is first ingested (but before when the job is scheduled), it will say "Complete" only to then change to 0%.

Reply:
Yeah, that kind of came to mind for me as well.  Though since philor's request was for "complete" when there are no jobs, I went this direction.  But perhaps the biggest thing is that "% - 0" isn't desired.  :)  

It feels to me like just leaving it blank if there's no jobs at all makes the most sense.  Let's go with that and see if we get complaints...  :)